concordia/admin/views.py

```diff
@@ -192,27 +208,30 @@ def admin_bulk_import_review(request):
                             continue
 
                         try:
+                            urls.append(url)
+                            url_counter = url_counter + 1
 
-                            count = event_loop.run_until_complete(
-                                import_item_count_from_url(url)
-                            )
-                            total_count = total_count + count
+                            if url_counter == 30:
```

Comment:
This could be simplified to `url_counter % 30 == 0` or `len(all_urls) % 30 == 1`